### ISSUE TITLE
globalDidAppear doesn't pass along passProps

### ISSUE THREAD

**Issue by Powersource** [staltz/cycle-native-navigation-web#2]
If I attach a .debug listener after https://gitlab.com/staltz/manyverse/-/blob/master/src/frontend/screens/central/private-tab/model.ts#L64 I get this logged when i enter the Conversation screen from the recipients-input screen

```js
{componentId: "Manyverse.Conversation---4", componentName: "Manyverse.Conversation"}
```

but the app also expects that object to normally contain the `passProps` key so I end up with an error

```
TypeError: Cannot read property 'recps' of undefined
    at Object.isNewConversationProps
```

**Comment by staltz** [comment]
Try version 0.5.0

**Comment by Powersource** [comment]
![image](https://user-images.githubusercontent.com/1427063/128389500-37db25c9-600a-4f89-a3c7-69f71b277d81.png)

**Comment by staltz** [comment]
It was a super obvious mistake, should be fixed now in 0.5.1

**Comment by Powersource** [comment]
heh, I thought it looked a bit weird that it wasn't returning anything
